Dockerfile

```diff
@@ -0,0 +1,61 @@
+FROM golang:1.13.3-alpine as builder
+
+WORKDIR /src
+
+RUN apk add --no-cache git
+
+ARG CADDY_SOURCE_VERSION=v2
+
+RUN git clone -b $CADDY_SOURCE_VERSION https://github.com/caddyserver/caddy.git --depth 1
+
+WORKDIR /src/caddy/cmd/caddy
+
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 \
+    go build -trimpath -tags netgo -ldflags '-extldflags "-static" -s -w' -o /usr/bin/caddy
+
+FROM alpine:3.10.3 AS alpine
+
+COPY build/files .
+COPY docker-entrypoint.sh /docker-entrypoint.sh
+
+COPY --from=builder /usr/bin/caddy /usr/bin/caddy
+COPY --from=builder /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs
+COPY --from=builder /etc/passwd /etc/passwd
+
+ARG VCS_REF
+ARG VERSION
+LABEL org.opencontainers.image.revision=$VCS_REF \
+    org.opencontainers.image.version=$VERSION \
```

Comment:
`ARG` only supports a single argument, unlike `LABELS` which does. The convention to have a single `LABEL` for multiple values dates back to when it caused the image to have more layers as a result. However that's no longer an issue, as of Docker 1.10: 

> Prior to Docker 1.10, it was recommended to combine all labels into a single `LABEL` instruction, to prevent extra layers from being created. This is no longer necessary, but combining labels is still supported.

_(from https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#label)_

Old habits die hard I guess - I can certainly split this into multiple `LABEL` instructions 😉